games-simulation/simgear/simgear-2018.2.2.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="Simulator Construction Gear"
+DESCRIPTION="SimGear is a set of open-source libraries designed \
+to be used as building blocks for quickly assembling 3D simulations, \
+games, and visualization applications."
+HOMEPAGE="https://www.flightgear.org/"
+COPYRIGHT="Curtis Olsen and others"
+LICENSE="LGPL-2.0 AND GPL-2.0 AND MIT"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/project/flightgear/release-2018.2/simgear-$portVersion.tar.bz2"
+CHECKSUM_SHA256="f61576bc36aae36f350154749df1cee396763604c06b8a71c4b50452d9151ce5"
+SOURCE_DIR="simgear-$portVersion"
+PATCHES="simgear-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	lib:simgear$secondaryArchSuffix = $portVersion

```

Comment:
Are you sure the recipe builds on x86_gcc2 primary arch, i.e. with gcc2 ?
* If it does, then:
```diff
 ARCHITECTURES="x86_gcc2 x86 x86_64"
-if [ "$targetArchitecture" = x86_gcc2 ]; then
-SECONDARY_ARCHITECTURES="x86"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
-fi
 
 PROVIDES="
+	simgear$secondaryArchSuffix = $portVersion
 	lib:simgear$secondaryArchSuffix = $portVersion
	devel:simgear$secondaryArchSuffix = $portVersion
	"
```
If it does not build with gcc2, then:
```diff
-ARCHITECTURES="x86_gcc2 x86 x86_64"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
 if [ "$targetArchitecture" = x86_gcc2 ]; then
 SECONDARY_ARCHITECTURES="x86"
 fi
 
 PROVIDES="
+	simgear$secondaryArchSuffix = $portVersion
 	lib:simgear$secondaryArchSuffix = $portVersion
	devel:simgear$secondaryArchSuffix = $portVersion
	"
```

Reply:
Fixed.